client-ts/signalr/build/process-dts.js

```diff
@@ -0,0 +1,8 @@
+// Edit the index.d.ts file to add the UMD export
+const fs = require('fs');
+const path = require('path');
+
+const target = path.resolve(__dirname, "..", "dist", "esm", "index.d.ts");
+
+let content = fs.readFileSync(target);
+fs.writeFileSync(target, content + "\r\nexport as namespace signalR;");
```

Comment:
This syntax (`export as namespace signalR`) allows the dts file to be used both when using it from a module (`import { HubConnection } from "@aspnet/signalr"`) and when using it from a "script" (a file with no `imports`, via `let c = new signalR.HubConnection`).